README.md

```diff
@@ -42,6 +42,21 @@ sudo apt install la-capitaine-icon-theme
 ```
 -->
 
+### Installation
+You'll need git to install this icon theme
+
+1. Open terminal and execute the following
+```
+git clone https://github.com/keeferrourke/la-capitaine-icon-theme.git ~/.icons/la-capitaine-icon-theme
+```
+
+2. Select this theme in tweak tool or execute the following
+```
+gsettings set org.gnome.desktop.interface icon-theme "la-capitaine-icon-theme"
+```
+
+3. Thats it you're done!
```

Comment:
This line is hella confusing considering the section that immediately follows is entitled "configuration". If that was it, there'd be nothing to configure!

Reply:
```
gsettings set org.gnome.desktop.interface icon-theme "la-capitaine-icon-theme"
```
wow, sorry bro because I'm a beginner for kali linux.
but the feature of above command is same as select this theme in tweak tool and I tested.